src/components/Home/NoticeFunSystem/NoticeFunSystem.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import * as styles from './NoticeFunSystem.style';
+
+interface SummaryText{
+  textContent : string;
+  date : string;
+}
+
+export interface NoticeFunSystemProps{
+  title : string;
+  summaries : SummaryText[];
+}
+
+
+const NoticeFunSystem:React.FC<NoticeFunSystemProps> = ({title, summaries}) => {
```

Comment:
```suggestion
const NoticeFunSystem = (props:NoticeFunSystemProps) => {
```
로 변경하고, 밑에서는 `props.title`로 접근해보는건 어떤가요? 생각해보고 코멘트 주세요!

Reply:
이런 방법도 있네요 감사합니다!